### PR TITLE
Docs: Improvements

### PR DESCRIPTION
**Pull request by hannahle**

### CODE REVIEW

File: docs/source/examples/docker_recipes.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Docker Recipes
+
+This page contains some recurrent implementation patterns used in Dockerfile when constructing workflows.
```
Comment (kennyworkman) [inline docs/source/examples/docker_recipes.md:3]: This page contains some common implementation patterns used in `Dockerfile`s when building images for workflows.

File: docs/source/index.md [hunk 2]
```diff
@@ -1,86 +1,62 @@
...
+* <b>[Concepts](./basics/what_is_a_workflow.md)</b> describes all important Latch SDK concepts.
+* <b>[Examples](./examples/workflows_examples.md)</b> show full examples of using Latch SDK for various bioinformatics pipelines.
+* <b>[Troubleshooting](./troubleshooting/troubleshooting)</b> provides a guide to debug common errors. 
+* <b>[Reference](./api/modules.rst)</b> contains detailed API and design documents.
+* <b>[Subcommands](./subcommands.md)</b> contains details about the Latch command line toolchain to register workflows and upload data to Latch.
+* Join the <a href="https://forms.gle/sCjr8tdjzx5HjVW27" target="_blank">SDK open-source community</a> on Slack!
 
 ---
 
-### Examples
```
Comment (kennyworkman) [inline docs/source/index.md:76]: i still want these on the front page, maybe link to the section page you made

File: docs/source/index.md [hunk 3]
```diff
@@ -1,86 +1,62 @@
...
-[here](https://console.latch.bio/workflows) and see your new workflow in your
-account.
+## Problem Latch SDK does not yet solve 
+* <b>Workflows chaining</b>: We'll aim to support easy installation and reuse of other Latch SDK workflows in your own workflow.  
+* <b>Workflows monitoring</b>: For batched runs of workflows, we'll aim to provide better dashboard, logs, traces, metrics, and alerting for observability. 
 
-If you are having issues with registration or have general questions, please
-file an issue on [github](https://github.com/latchbio/latch).
+## What Latch SDK is not
+* <b>A pure workflow orchestration engine</b>: There are many popular workflow orchestration engines, such as Nextflow or Snakemake, that can be run locally from a bioinformatician's machine. Although workflow orchestration is a feature of Latch SDK, Latch is fundamentally a cloud-native solution. Latch SDK solves the problems of automating pipelines by providing a robust and scalable cloud infrastructure and asssociated no-code interfaces out of the box. You can also easily bring existing workflow script of any language to Latch (See examples [here](./examples/)).
```
Comment (kennyworkman) [inline docs/source/index.md:42]: "Latch is fundamentally a cloud-native solution." -> "Latch also provides managed infrastructure and no-code interface generation."

File: docs/source/index.md [hunk 4]
```diff
@@ -1,86 +1,62 @@
...
 
-This might take 3-10 minutes depending on your network connection. (Subsequent
-registers will complete in seconds by reusing the image layers from this initial
-register.) The registration process will:
+<b>Bioinformatics tools face the challenges of irreproducibility.</b> The lack of proper versioning and dependencies management results in a long tail of poorly documented and unusable bioinformatics software tools.
 
-* Build a docker image containing your workflow code
-* Serialize your code and register it with your LatchBio account
-* Push your docker image to a managed container registry
+Latch SDK containerizes and versions the code in the background each time a workflow is registered to the Latch platform. Container images are constructed by parsing the python runtime for imported libraries or defined using a [Dockerfile](https://docs.docker.com/engine/reference/builder/). Similarly, versions are user-specified as any unique plaintext string. This behavior is a strict requirement of the toolchain and gives us remarkable guarantees with respect to code reproducibility, portability and scalability.
```
Comment (kennyworkman) [inline docs/source/index.md:35]: get rid of "by parsing the python runtime for imported libraries or" (no longer true)

File: docs/source/getting_started/quick_start.md [hunk 5]
```diff
@@ -0,0 +1,364 @@
+# Quick Start
+
+In this tutorial, we will: 
+* Write a workflow to sort and assemble COVID genomes
```
Comment (kennyworkman) [inline docs/source/getting_started/quick_start.md:4]: COVID sequencing data

File: docs/source/getting_started/quick_start.md [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+* Customize a front-end interface for our workflow
+* Define the necessary compute and resource requirements
+* Upload the workflow onto Latch
+* Test our workflow from a user-friendly interface 
+
+After going through the tutorial, you should have a conceptual understanding of how to translate a set of command lines into a Latch workflow and how to generate an interactive interface for that workflow.
```
Comment (kennyworkman) [inline docs/source/getting_started/quick_start.md:10]: "you should have a conceptual understanding of how to translate a set of command lines into a Latch workflow" -> "you should have a strong understanding of how to translate a handful of python functions into a Latch workflow"

File: docs/source/getting_started/quick_start.md [hunk 7]
```diff
@@ -0,0 +1,364 @@
...
+        "-S",
+        str(sam_file),
+    ]
+
+    subprocess.run(_bowtie2_cmd)
+
+    return LatchFile(str(sam_file), "latch:///covid_assembly.sam")
+```
+
+<i>1.1 Decorating a task</i>
```
Comment (kennyworkman) [inline docs/source/getting_started/quick_start.md:105]: replace all html-based markup with markdown (`**` or `_`) for consistency

Comment (kennyworkman) [reply docs/source/getting_started/quick_start.md:105]: also these should be markdown headings

**Review by kennyworkman — CHANGES_REQUESTED** [review]

File: docs/source/index.md (hunk 2 above)
Comment (hannahle) [reply docs/source/index.md:76]: I linked that in Next steps. Do you want to make it big and center?

**Pull request merged**